app/src/androidTest/java/com/sdp13epfl2021/projmag/UserTypeChoiceTest.kt

```diff
@@ -21,84 +21,29 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
-@LargeTest
-@RunWith(AndroidJUnit4::class)
+
 class UserTypeChoiceTest {
 
-    @Rule
-    @JvmField
+    @get:Rule
     var mActivityTestRule = ActivityTestRule(UserTypeChoice::class.java)
 
     @Test
-    fun userTypeChoiceTest() {
-        val materialRadioButton = onView(
-            allOf(
-                withId(R.id.radioProfessorType), withText("Professor"),
-                childAtPosition(
-                    childAtPosition(
-                        withClassName(`is`("androidx.constraintlayout.widget.ConstraintLayout")),
-                        0
-                    ),
-                    0
-                ),
-                isDisplayed()
-            )
-        )
-        materialRadioButton.perform(click())
+    fun userProf() {
+        onView(withId(R.id.radioProfessorType)).perform(click())
         assertTrue(UserTypeChoice.isProfessor)
+    }
 
```

Comment:
these kind of tests using singletons are critcized in the code review should be changed, now or once we introduced dependency injections